lib/cucumber/formatter/json.rb

```diff
@@ -257,7 +259,8 @@ def background(background)
             name: background.name,
             description: value_or_empty_string(background.description),
             line: background.location.line,
-            type: 'background'
+            type: 'background',
+            steps: []
```

Comment:
Do we want to just be extra safe and make line255/257 and line266/268 a memo `||=` instead. As this could potentially overwrite the background steps with a set of empty steps?

Reply:
Actually this is first instantiation of the background and test_case hashes here so it is actually safe.

```ruby
        @background_hash = {
            # snip...
            type: 'background',
            steps: []
          }
# ...
        @test_case_hash = {
            # snip...
            type: 'scenario',
            steps: []
          }
```

Reply:
So I think I've spotted how this works, but this definitely could use a bit of love and care.